docs/events.md

```diff
@@ -54,26 +54,26 @@ in case the event sends additional arguments in the future.
 
 ## List of emitted events
 
-| Event                 | Mutations that can emit this event                                | Arguments                      |
+| Event                     | Mutations that can emit this event                                | Arguments                      |
 | --------------------- | ----------------------------------------------------------------- | ------------------------------ |
-| `created_work_item`   | `CreateWorkItem`, `SaveWorkItem`, `StartCase`, `CompleteWorkItem` | `work_item`, `user`, `context` |
-| `completed_work_item` | `CompleteWorkItem`                                                | `work_item`, `user`, `context` |
-| `canceled_work_item`  | `CancelCase`, `CancelWorkItem`                                    | `work_item`, `user`, `context` |
-| `skipped_work_item`   | `SkipWorkItem`                                                    | `work_item`, `user`, `context` |
-| `suspended_work_item` | `SuspendWorkItem`                                                 | `work_item`, `user`, `context` |
-| `resumed_work_item`   | `ResumeWorkItem`                                                  | `work_item`, `user`, `context` |
-| `created_case`        | `SaveCase`, `StartCase`                                           | `case`, `user`, `context`      |
-| `completed_case`      | `CompleteWorkItem`                                                | `case`, `user`, `context`      |
-| `canceled_case`       | `CancelCase`                                                      | `case`, `user`, `context`      |
-| `suspended_case`      | `SuspendCase`                                                     | `case`, `user`, `context`      |
-| `resumed_case`        | `ResumeCase`                                                      | `case`, `user`, `context`      |
+| `post_create_work_item`   | `CreateWorkItem`, `SaveWorkItem`, `StartCase`, `CompleteWorkItem` | `work_item`, `user`, `context` |
+| `post_complete_work_item` | `CompleteWorkItem`                                                | `work_item`, `user`, `context` |
```

Comment:
I think `pre_complete_work_item` is missing here.

Reply:
updated the docs